backend/backend/urls.py

```diff
@@ -68,6 +72,11 @@
 	path('match/', MatchView.as_view()),
 	path('matches/', MatchHistory.as_view()),
 
+	# Player views
+	path('players/', PlayerListView.as_view()),
+	path('players/<int:player_id>', PlayerDetailView.as_view()),
```

Comment:
What happened to the PR that was fixing those?

Reply:
what do you mean? Fix what?